### ISSUE TITLE
Pretraining T5-v1_1 on Flax

### ISSUE THREAD

**Issue by peregilk** [huggingface/transformers#13287]
@patrickvonplaten 

In the[Flax tutorial](https://github.com/huggingface/transformers/tree/master/examples/flax/language-modeling) it is recommended loading the config from t5-v1_1-base when pretraining, using:

`config = T5Config.from_pretrained("google/t5-v1_1-base", vocab_size=tokenizer.get_vocab_size())`

This basically copies this [config](https://huggingface.co/google/t5-v1_1-base/blob/main/config.json)

It seems like this is tuned for finetuning, since it has the line:
'"dropout_rate": 0.1'

Google [states](https://huggingface.co/google/t5-v1_1-base) that _"Dropout was turned off in pre-training (quality win). Dropout should be re-enabled during fine-tuning."._ Should this be modified for pretraining?

Google also state that "no parameter sharing between embedding and classifier layer". How is this achieved?
